include/LightGBM/config.h

```diff
@@ -684,6 +684,12 @@ struct Config {
 
   #pragma region Predict Parameters
 
+  // [no-save]
+  // desc = used only in ``prediction`` task
+  // desc = used to specify from which iteration to start the prediction
+  // desc = ``<= 0`` means from the first iteration
+  int start_iteration_predict = 0;
```

Comment:
Please run `python helpers/parameter_generator.py` to regenerate docs and pass CI test.